src/herbie.rkt

```diff
@@ -12,6 +13,18 @@
 (define (string->thread-count th)
   (match th ["no" #f] ["yes" (max (- (processor-count) 1) 1)] [_ (string->number th)]))
 
+(define (operator-pruning)
```

Comment:
This is a pretty strange name. Normally an action would be named something like `prune-operators!` but in this case you really want to capture the idea of *possibly* warning about operators—I'd call it something like `check-operator-fallbacks!`.